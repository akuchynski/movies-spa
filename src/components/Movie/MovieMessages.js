import React from "react";

export const MovieMessages = ({ isLoading, errorMessage }) => {
    return (
        <>
            {isLoading && <h3>Loading movies...</h3>}
            {errorMessage && <h3>{errorMessage}</h3>}
        </>
    );
};