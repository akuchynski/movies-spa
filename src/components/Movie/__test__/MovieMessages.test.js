import React from "react";
import { render } from "@testing-library/react";
import { MovieMessages } from "../MovieMessages";

describe('movie messages tests', () => {
    it('should render movie messages component', () => {
        const { asFragment } = render(<MovieMessages isLoading={true}/>);
        expect(asFragment()).toMatchSnapshot();
    });
});