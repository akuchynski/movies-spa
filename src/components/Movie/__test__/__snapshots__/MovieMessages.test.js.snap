// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movie messages tests should render movie messages component 1`] = `
<DocumentFragment>
  <h3>
    Loading movies...
  </h3>
</DocumentFragment>
`;
